src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("successful fetched data from DB. Params: userId = {}", userId);
```

Comment:
The log message should be more descriptive and use the correct method name. For instance, 'Successfully fetched products from the shopping cart for user with id={}'.